hack/test-go.sh

```diff
@@ -181,7 +181,7 @@ else
             -o -path "${kubernetes_path}/test/e2e"                                                    \
             -o -path "${kubernetes_path}/cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned" \
           \) -prune                                                                                   \
-        \) -name '*_test.go' -print0 | xargs -0n1 dirname -z | sort -uz | xargs -0n1 printf "${OS_GO_PACKAGE}/%s\n")"
+        \) -name '*_test.go' | xargs -n1 dirname | sort -u | xargs -n1 printf "${OS_GO_PACKAGE}/%s\n")"
```

Comment:
Yes, this does. The interesting thing is if you look at this pipe before my changes, some of the middle members of the pipe never honored the `-print0` nil delimiter, which I noticed and fixed. This is what the `-z` flag for `dirname` was accomplishing for us. Therefore, we were never actually supporting the case where a filename contained spaces or newlines anyway.